### DISCUSSION TITLE
How tell vcpkg to use -isystem for package include directory

### DISCUSSION THREAD

**Discussion by quyykk** [microsoft/vcpkg#25603] [Q&A]
Packages can be considered system libraries and as such they should use the `-isystem` for gcc/clang instead of `-I`. I encountered this because clang emits a certain warning inside SDL2 files which is not ideal. Adding the include directories with `-isystem` would fix this, instead of `-I` which is how they are currently specified.

I'm using vcpkg as submodule with the CMake integration.

Is there any way to change this?

**Comment by JackBoosY** [comment]
Create a new toolchain file or modify the current toolchain settings.

**Reply by a4z** [reply]
I do not have the feeling this is a useful answer 

how to modify, what is required
